wbooks/src/constants/dimensions.ts

```diff
@@ -0,0 +1,4 @@
+export const DIMENSIONS = {
+  width: 25,
+  height: 25
+};
```

Comment:
i didn't mean to extract them so far beyond. You just only needed to extract a constant value like
`const SIZE = 25;`
inside the style file, this is not a global value for all styles in your app